connector/src/main/scala/com/basho/riak/spark/writer/WriteConf.scala

```diff
@@ -31,4 +37,9 @@ object WriteConf {
       writeQuorum = conf.getInt(WriteQuorumProperty, DefaultWriteQuorum)
     )
   }
+  
+  def fromOptions(options: Map[String, String], conf: SparkConf): WriteConf = {
```

Comment:
It should be renamed to apply(), scaladoc should be added.


Also please rename fromSparkConf() to apply() and add scaladoc